ludwig/features/binary_feature.py

```diff
@@ -284,8 +290,14 @@ def loss_kwargs(self):
             confidence_penalty=self.loss["confidence_penalty"],
         )
 
+    def create_calibration_module(self, feature) -> torch.nn.Module:
+        calibration_type = feature.get("calibration")
+        if calibration_type == "temperature_scaling":
+            return calibration.TemperatureScaling(binary=True)
+        return None
```

Comment:
here and in the category feature too we can create a registry like we do for most other things and use the functions to get values from registries because they already deal with wrong keys by returning errors. That would help 1) consistency and 2) avoiding silent errors like misspelled calibration type names.